test/sync/src/commonMain/kotlin/io/realm/kotlin/entities/sync/SyncObjectWithAllTypes.kt

```diff
@@ -118,12 +121,16 @@ class SyncObjectWithAllTypes : RealmObject {
                             RealmStorageType.INT -> {
                                 Pair(
                                     { obj: SyncObjectWithAllTypes ->
+                                        obj.mutableRealmIntField = MutableRealmInt.of(42)
+                                        obj.mutableRealmIntNullableField = null
                                         obj.intField = 42
                                         obj.intNullableField = 42
                                         obj.intRealmList = realmListOf(42)
                                         obj.intRealmSet = realmSetOf(42)
                                     },
                                     { obj: SyncObjectWithAllTypes ->
+                                        assertEquals(MutableRealmInt.of(42), obj.mutableRealmIntField)
```

Comment:
Also missing tests of `increment` and `decrement` working across Sync. Maybe not that multiple clients converge, but at least that increment and decrement actually works when hitting the server.

Reply:
Sure, I've added a new specific test as integrating it in the roundtrip test we have is a bit of a hassle.